rules-tests/Php80/Rector/Identical/StrEndsWithRector/Fixture/substr_not_prefix.php.inc

```diff
@@ -20,7 +21,7 @@ class SubstrNotPrefix
 {
     public function run()
     {
-        $isMatch = !str_ends_with($haystack, $needle);
+        $isNotMatch = !str_ends_with($haystack, $needle);
```

Comment:
I think you can add another line for flip check like other fixtures, so no need another file:

```suggestion
        $isNotMatch = !str_ends_with($haystack, $needle);
        $isNotMatch = !str_ends_with($haystack, $needle);
```